### ISSUE TITLE
World Terrain is rendered without texture (Unity Quickstart)

### ISSUE THREAD

**Issue by csciguy8** [CesiumGS/cesium-unity#313]
1. Install Unity Hub 3.4.2 and Unity 2021.3.25f1.

2. Follow the 'Cesium for Unity Quickstart" guide,
https://cesium.com/learn/unity/unity-quickstart/

3. When creating the new project, use the "3D URP" template. 

4. When installing Cesium for Unity, select version 1.2.0.

6. Continue with the tutorial until you add "Cesium World Terrain + Bing Maps Aerial imagery".

Notice that the world terrain geometry appears, but there is no texture. It is flat grey.

Also of note, if you repeat the same steps, but in step 3 use the "3D HDRP" template instead, everything seems to work fine.

**Comment by j9liu** [comment]
I was able to reproduce this too, with some notes:

- The empty 3D URP scene works, which leads me to think that something specific in the sample scene breaks our plugin somehow?
- Refreshing the tileset does nothing.
- With "Show Tiles In Hierarchy" enabled, the materials on the individual primitive objects look correct. I.e., the textures show up correctly in the material preview window.
- Forcing the shader graph to recompile (by selecting the render pipeline in the editor, and clicking "Save Asset") doesn't help
- If I assign the tileset an opaque material, it doesn't change the appearance at all.
- I tried adding Cesium OSM Buildings to the scene as well. The geometry was visible but all buildings were rendered with the flat grey color.

**Comment by kring** [comment]
Just to exclude the simple stuff: it's not flat white because Unity is still in the process of compiling shaders, is it?

**Comment by kring** [comment]
Good point, I'm thinking of Unreal, which shows a flat grey/white while the correct shader is missing.

**Comment by AvinoamHeroicRealm** [comment]
Hi did you manage to solve this? I can't seem to upgrade my cesium materials, and so the globe's stuck on this bright cyan colour

**Comment by AvinoamHeroicRealm** [comment]
> Hi did you manage to solve this? I can't seem to upgrade my cesium materials, and so the globe's stuck on this bright cyan colour

Ok I've managed to solve this.. there seem to have been an error in the shader graph shaders. What worked for me (for some reason) was to open them in the shader graph editor and simply click save asset. unity apparently fixed them behind the scenes and that worked for me
